vm-virtio/src/vhost_user/handler.rs

```diff
@@ -1,9 +1,5 @@
-// Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
-// SPDX-License-Identifier: Apache-2.0
-//
-// Portions Copyright 2017 The Chromium OS Authors. All rights reserved.
-// Use of this source code is governed by a BSD-style license that can be
-// found in the THIRD-PARTY file.
+// Copyright 2019 Intel Corporation. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 AND BSD-3-Clause
 
```

Comment:
Squash this changes to commit 3081c88, thanks

Reply:
Yes please squash :)